### PR TITLE
Bench project blog

### PR DESCRIPTION
**Pull request by GospelBeats**

### CODE REVIEW

File: posts/advancing from micro services to including micro frontends and how this leads to vertical teams.md
Comment (dsharpe) [inline posts/advancing from micro services to including micro frontends and how this leads to vertical teams.md:94]:
```suggestion
Are you still trying to expand your enterprise to a more loosely-coupled platform? Make sure to extend your production environment past just microservices, and all the components of your application are scalable to evolve with customer needs. To add or discover more about micro frontends, check out: https://martinfowler.com/articles/micro-frontends.html. You can also learn about Entando at https://entando.com if you are interested in using a platform that utilizes micro frontends as one of its fundamental development elements. Stay caught up as technology changes and peruse the other [blogs](https://blog.ippon.tech) and [articles](https://medium.com/ippon-technologies-usa) Ippon has to offer.
```

File: posts/advancing from micro services to including micro frontends and how this leads to vertical teams.md
Comment (codyfrenzel) [inline posts/advancing from micro services to including micro frontends and how this leads to vertical teams.md:17]: loosely-coupled architecture*

File: posts/advancing from micro services to including micro frontends and how this leads to vertical teams.md
Comment (codyfrenzel) [inline posts/advancing from micro services to including micro frontends and how this leads to vertical teams.md:40]: I feel like the microservice section is talking about a MFE instead of an API.

File: posts/advancing from micro services to including micro frontends and how this leads to vertical teams.md
Comment (GospelBeats) [reply posts/advancing from micro services to including micro frontends and how this leads to vertical teams.md:57]: Replace a few "micro frontends" with "MFE", but left some for SEO purposes when mention later in the blog

File: posts/advancing from micro services to including micro frontends and how this leads to vertical teams.md
Comment (GospelBeats) [reply posts/advancing from micro services to including micro frontends and how this leads to vertical teams.md:41]: made the suggested changes

File: posts/advancing from micro services to including micro frontends and how this leads to vertical teams.md
Comment (GospelBeats) [reply posts/advancing from micro services to including micro frontends and how this leads to vertical teams.md]: made the suggested changes

File: posts/advancing from micro services to including micro frontends and how this leads to vertical teams.md
Comment (GospelBeats) [reply posts/advancing from micro services to including micro frontends and how this leads to vertical teams.md]: made the suggested changes

**Comment by GospelBeats** [comment]
All feedback, suggestions, comments have been addressed, Thanks for you guys input! I guess I'm waiting for another final review?

**Review by dsharpe — APPROVED** [review]

**Pull request merged**
